config/install/migrate_plus.migration.node.yml

```diff
@@ -40,29 +39,78 @@ process:
   # in.
   field_alternative_title:
     plugin: skip_on_empty
-    source: subtitle 
+    source: subtitle
     method: process
 
   field_description: description
 
   # Dates are EDTF strings
   field_edtf_date: issued
-    
-  # Make the object an 'Image'
+
+  _mimetype:
+    -
+      # Get a mimetype (eg image/jpeg) from mimetype process plugin.
+      source: file
+      plugin: file_mimetype
+
+  _model:
+    -
+      # Get a mimetype (eg image/jpeg) from mimetype process plugin.
+      source: file
+      plugin: file_mimetype
```

Comment:
Is this necessary? Below you call to the above `_mimetype`, but my eyes might be getting tired.